src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -320,7 +320,8 @@ def init_sync_d_queue_tables(db_conn):
         queue_stat_name = queue_table(sai_id)
```

Comment:
>queue_stat_name [](start = 8, length = 15)

SAI Object ID is unique in this `db_conn`. So the original implementation of `init_sync_d_queue_tables` makes sense.

Let's block this PR and continue exploration the dbs dict strategy, as also discussed in another PR ([https://github.com/Azure/sonic-snmpagent/pull/137](https://github.com/Azure/sonic-snmpagent/pull/137))

Reply:
As the data will eventually merged together , we have to make sure that the keys are unique in these dictionaries.
So in maps which use SAI ID as key, modified it to use a combination of if_idx and SAI ID.